lib/signaling/v2/peerconnectionmanager.js

```diff
@@ -188,6 +188,8 @@ class PeerConnectionManager extends QueueingEventEmitter {
       peerConnection.on('stateChanged', function stateChanged(state) {
         if (state === 'closed') {
           peerConnection.removeListener('stateChanged', stateChanged);
+          self._dataTrackSenders.forEach(peerConnection.removeDataTrackSender, peerConnection);
+          self._mediaTrackSenders.forEach(peerConnection.removeMediaTrackSender, peerConnection);
```

Comment:
I think making an arrow function makes it more readable.
```suggestion
          self._dataTrackSenders.forEach(sender => peerConnection.removeDataTrackSender(sender));
          self._mediaTrackSenders.forEach(sender => peerConnection.removeMediaTrackSender(sender));
```